packages/dev/core/src/Engines/WebGPU/webgpuPipelineContext.ts

```diff
@@ -85,12 +85,6 @@ export class WebGPUPipelineContext implements IPipelineContext {
     ) {
         const engine = this.engine;
 
-        // Prevent Memory Leak by reducing the number of string, refer to the string instead of copy.
-        effect._fragmentSourceCode = "";

```

Comment:
I remember this was crashing on the forest demo for instance if all shaders where a bit different as we where exploding the heap size with share code. Is this now somewhere else ?

Reply:
If we want to manage context loss in WebGPU, we need to keep the source code, so that we can recreate the effects... Just in case, I've added some code to delete strings if we don't have to deal with context loss.